pages/about.html

```diff
@@ -4,9 +4,32 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="../styles/style.css" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
+    <header>
+      <nav id="navigation">
+        <a href="index.html" >Home</a>
+        <a href="portfolio.html">My portfolio</a>
+      </nav>
+      <h1>About Ja</h1>
+    </header>
+    <h2 id="projects">Introduction</h2>
+    <section class="about_container">
+      <p id="introduction">Hi! My name is Ja and I am software engineer in
+        training! Fun right? I know! My days are filled with studying but in my spare time, I love to play with
+        my kitty("Omi") and make jewelry for my small business, Pocketofjade! 
+        My absolute favorite things to do are cooking and exploring Las Vegas with my girlfriend.
+      </p>
+      <div id="cat"><img src="../images/kitty.jpg" alt="Omi"></div>
+      <!-- <div id="jewels"><img src="../images/jewels.JPG" alt="H=Jewelry"></div> -->
+      <div id="jewels2"><img src="../images/jewels2.JPG" alt="More Jewelry"></div>
+      <div id="me"><img src="../images/me.jpg" alt="Me"></div>
+    </section>
+    <footer>
+      <p><a href="mailto:jacqueline@gmail.com">Contact me</a><br> &copy; 2022 Ja Hopkins</p>
```

Comment:
Try to avoid using `br` tags to line wrap. Prefer modifying the display of toher tags. For instance, if the `a` with your email had a `display: block;` that would force the copyright line to wrap to the next line. `br` is more for cases when we have a logical chunk of text (so wrapping individual bits in tags might be arbitrary), but we want to preserve something about where line breaks occur. The typical example is a poem. We probably wouldn't want to wrap each line with a tag (though that would also work). Rather, we might wrpa each stanza in a `p` and the use `br` to break the lines without relying on automatic line wrapping.